cob_bringup/drivers/hokuyo.launch

```diff
@@ -9,9 +9,7 @@
 	<rosparam ns="laser_$(arg name)" command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/laser_$(arg name).yaml"/>
 
 	<!-- start laser driver -->
-	<node ns="laser_$(arg name)" pkg="hokuyo_node" type="hokuyo_node" name="laser_$(arg name)" respawn="false" output="screen">
-		<remap from="scan" to="/scan_$(arg name)_raw"/>
-	</node>
+	<node ns="laser_$(arg name)" pkg="hokuyo_node" type="hokuyo_node" name="laser_$(arg name)" respawn="false" output="screen"/>
```

Comment:
As soon as we agree on what topic name to use for laser scanners, this feature will be proposed in a separate PR and reduced from this PR...